api_tests/fixtures/populate.coffee

```diff
@@ -3,21 +3,21 @@ __ = CONFIG.universalPath
 _ = __.require 'builders', 'utils'
 { Promise } = __.require 'lib', 'promises'
 { createUser } = require './users'
-{ createRandomizedItem } = require './items'
+{ createRandomizedItems } = require './items'
+randomString = __.require 'lib', './utils/random_string'
 
-module.exports = (params) ->
-  { usersCount } = params
-  promiseArray = [1..usersCount].map ->
-    createUserWithItems(params)
-  Promise.all promiseArray
+populatePromise = null
+usersCount = 2
+publicItemsPerUser = 16
```

Comment:
maybe 3 items per user by default is enough, less than 16 would be better

Reply:
I choose `2` and `16` to match the parameters from [`recent_public_items`](https://github.com/inventaire/inventaire/pull/109/files#diff-2882db99c0bf7b4963427781e781053b) tests, as the populate function doesn't take parameters anymore, but once this branch is merge, you are welcome to rebase your branch on top of it and customize it to fit your tests